documentation/api/io.strimzi.api.kafka.model.KafkaMirrorMaker2ClusterSpec.adoc

```diff
@@ -1,18 +1,9 @@
-Use the three allowed `ssl` configuration options to run external listeners with a specific _cipher suite_ for a TLS version.
-A _cipher suite_ combines algorithms for secure connection and data transfer.
+Use `KafkaMirrorMaker2ClusterSpec` properties to configure Kafka clusters for mirroring.
 
-.Example SSL configuration
-[source,yaml,subs="attributes+"]
-----
-spec:
-  clusters:
-    config:
-      ssl.cipher.suites: "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384" <1>
-      ssl.enabled.protocols: "TLSv1.2" <2>
-      ssl.protocol: "TLSv1.2" <3>
-----
-<1> The cipher suite for TLS using a combination of `ECDHE` key exchange mechanism, `RSA` authentication algorithm,
-`AES` bulk encyption algorithm and `SHA384` MAC algorithm.
-<2> The SSl protocol `TLSv1.2` is enabled.
-<3> Specifies the `TLSv1.2` protocol to generate the SSL context.
-Allowed values are `TLSv1.1` and `TLSv1.2`.
+[id='property-mirrormaker2-config-{context}']
+.`config`
+Use the `config` properties to configure Kafka options.
+
+Standard Apache Kafka configuration may be provided, xref:assembly-kafka-connect-configuration-deployment-configuration-kafka-connect[restricted to those properties not managed directly by Strimzi].
+
+To run external listeners with a specific _cipher suite_ for a TLS version, you can xref:con-common-configuration-cipher-reference[configure allowed `ssl` properties].
```

Comment:
This does not configure listeners but clients.

Reply:
I changed to _For client connection using a specific cipher suite_ and assumed we needed this change wherever else this sentence was used.